components/frontend/src/measurement/MeasurementValue.js

```diff
@@ -1,39 +1,91 @@
+import "./MeasurementValue.css"
+
 import { bool, string } from "prop-types"
 import { useContext } from "react"
-import { Icon } from "semantic-ui-react"
 
 import { DataModel } from "../context/DataModel"
-import { Label, Popup } from "../semantic_ui_react_wrappers"
-import { datePropType, metricPropType } from "../sharedPropTypes"
+import { Icon, Label, Message, Popup } from "../semantic_ui_react_wrappers"
+import { datePropType, measurementPropType, metricPropType } from "../sharedPropTypes"
+import { SOURCE_ENTITY_STATUS_NAME } from "../source/source_entity_status"
 import {
     formatMetricValue,
     getMetricScale,
+    getMetricUnit,
     getMetricValue,
     isMeasurementRequested,
     MILLISECONDS_PER_HOUR,
+    sum,
 } from "../utils"
 import { TimeAgoWithDate } from "../widgets/TimeAgoWithDate"
 import { WarningMessage } from "../widgets/WarningMessage"
 
-function measurementValueLabel(stale, updating, value) {
+function measurementValueLabel(hasIgnoredEntities, stale, updating, value) {
+    const ignoredEntitiesIcon = hasIgnoredEntities ? <Icon name="hide" /> : null
     if (stale) {
-        return <Label color="red">{value}</Label>
+        return (
+            <Label color="red">
+                {ignoredEntitiesIcon} {value}
+            </Label>
+        )
     }
     if (updating) {
         return (
             <Label color="yellow">
-                <Icon loading name="hourglass" /> {value}
+                <Icon loading name="hourglass" /> {ignoredEntitiesIcon} {value}
```

Comment:
This seems a bit like code duplication, should we not create a helper method to check if it needs to wrap the content in colour yellow with hourglass icon?